spec/hydra/works/services/add_file_to_file_set_spec.rb

```diff
@@ -67,14 +67,6 @@
     end
   end
 
-  context 'file responds to :path but not to :mime_type nor :original_name' do
```

Comment:
Is it worth changing this test to just make sure both of the new service objects are called (but otherwise defer testing said objects to the new specs below)?